api/users/pagination.py

```diff
@@ -0,0 +1,60 @@
+from collections import OrderedDict
+from django.core.urlresolvers import reverse
+
+from api.base.pagination import JSONAPIPagination
+from rest_framework.response import Response
+
+class UserNodeLogPagination(JSONAPIPagination):
+
+    node_log_aggregates = None
+
+    def get_paginated_response(self, data):
```

Comment:
We can probably keep from repeating some of the code in `get_pagingated_response`.  I would add a method to the original pagination class we're using,  JSONAPIPagination, called `get_response_dict` or similar, which just returns the basic paginated response that we use in the majority of the api:
```
def get_response_dict(self, data, url):
        return OrderedDict([
            ('data', data),
            ('links', OrderedDict([
                ('first', self.get_first_real_link(url)),
                ('last', self.get_last_real_link(url)),
                ('prev', self.get_previous_real_link(url)),
                ('next', self.get_next_real_link(url)),
                ('meta', OrderedDict([
                    ('total', self.page.paginator.count),
                    ('per_page', self.page.paginator.per_page),
                ]))
            ])),
        ])
```
Then you call this method in JSONAPIPagination > get_paginated_response.  

Finally, instead of overriding the entirety of `get_paginated_response` in users/pagination, you can override just the `get_response_dict` using the same if/else logic you're already using: 

```
if self.request.query_params.get('aggregate'):
    ...return your dictionary with the aggregate info/last_log_date
else:
   return super(UserNodeLogPagination, self).get_response_dict(data, url)
```